README.md

```diff
@@ -1,2 +1,15 @@
-# newcomers-only-app
-1st app by RGSoC team $we init
+# FIRST-TIMERS ONLY BOT
+
+## Statement:
+
+A bot that ensures exactly one newcomer issue is assigned per person so that it’s fair for all newcomers. Additionally, it encourages the no-more-newcomers to take up challenging issues once they’re comfortable with the workflow after having tackled a newcomer issue.
+
+## Algorithm :
+
+1. Whenever a user comments on any issue, it will be detected using “issue_comment” event.
+2. The label of the issue is returned in the payload response of the said event.
+3. We will create a configurable json file containing an array of labels(easy-pick, first-timers, good-first-pr ) used for tagging new comer issues.
```

Comment:
I would suggest to follow the recommendation from firsttimersonly.com and look for the label [`first-timers-only`](https://github.com/search?q=label%3Afirst-timers-only&state=open&type=Issues). You can make it configurable later, but maybe it’s not even necessary.